### PR TITLE
add support for mqtt client TLS authentication

### PR DESCRIPTION
**Pull request by carlospsikick**
add new cli arguments to support connection with CA, Cert and Key files.

### CODE REVIEW

File: worker.go [hunk 1]
```diff
@@ -35,6 +37,9 @@ type Worker struct {
 	Retained            bool
 	PublisherQoS        byte
 	SubscriberQoS       byte
+	CA					[]byte
```
Comment (hikhvar) [inline worker.go:40]: can you please run `go fmt` ?

File: worker.go [hunk 2]
```diff
@@ -43,6 +48,39 @@ func setSkipTLS(o *mqtt.ClientOptions) {
...
+		RootCAs: certpool,
+		// ClientAuth = whether to request cert from server.
+		// Since the server is set up for SSL, this happens
+		// anyways.
+		ClientAuth: tls.NoClientCert,
+		// ClientCAs = certs used to validate client cert.
+		ClientCAs: nil,
+		// InsecureSkipVerify = verify that cert contents
+		// match server. IP matches what is in cert etc.
+		InsecureSkipVerify: true,
```
Comment (hikhvar) [inline worker.go:78]: This should be set to `false`. Setting this to `true` has the effect that the stresser will accept any certificate presented by the server.

**Comment by hikhvar** [comment]
Fixes #33

**Comment by carlospsikick** [comment]
Good catch. I Added the fixes.

File: main.go [hunk 3]
```diff
@@ -144,6 +175,29 @@ func main() {
...
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+		key, err = ioutil.ReadFile(*argKey)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+		*argSkipTLSVerification = false
```
Comment (hikhvar) [inline main.go:199]: This change does not have the intended effect. Your TLs config is added to the worker options after this flag is copied to the publisher options. Please just set InsecureSkipVerify to false as written in my previous comment.

Comment (carlospsikick) [reply main.go]: my bad, I misunderstood the request.

**Pull request merged**